examples/LiftSim_demo/rl_benchmark/agent.py

```diff
@@ -0,0 +1,121 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import numpy as np
+import numpy.random as random
+import paddle.fluid as fluid
+from parl import layers
+from parl import Agent
+from parl.utils import get_gpu_count
+
+
+class ElevatorAgent(Agent):
+    def __init__(self, algorithm, obs_dim, action_dim):
+        self._action_dim = action_dim
+        self._obs_dim = obs_dim
+        self._update_target_steps = 1000
+
+        self._global_step = 0
+        self.exploration_ratio = 0.9
+        self.exploration_decre = 1e-7
+        self.exploration_min = 0.1
+        super(ElevatorAgent, self).__init__(algorithm)
+
+        use_cuda = True if self.gpu_id >= 0 else False
```

Comment:
The attribute `gpu_id` of `parl.Agent` is deprecated, pls use our new API:  https://github.com/PaddlePaddle/PARL/blob/b29a1ec13df4f69c1b758db84c71bb81925ea9c8/examples/GA3C/atari_agent.py#L51